src/AppBundle/Entity/Adherent.php

```diff
@@ -226,6 +232,31 @@ public function changePassword(string $newPassword)
         $this->password = $newPassword;
     }
 
+    public function getEmailSubscriptions(): array
+    {
+        return $this->emailSubscriptions;
+    }
+
+    public function setEmailSubscriptions(array $emailSubscriptions)
+    {
+        $this->emailSubscriptions = $emailSubscriptions;
```

Comment:
We should alter 3 flags (properties) in this class based on what we have in the array.